Setup/Migrations/ReplaceDpzRange.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelNL\Magento\Setup\Migrations;
+
+use Magento\Framework\App\ObjectManager;
+use Magento\Framework\Setup\SchemaSetupInterface;
+use MyParcelNL\Magento\Setup\QueryBuilder;
+
+class ReplaceDpzRange
+{
+    /** @var string $pathName */
+    private string $pathName = 'myparcelnl_magento_postnl_settings/digital_stamp/active';
+
+    /** @var \MyParcelNL\Magento\Setup\QueryBuilder $queryBuilder */
+    private QueryBuilder $queryBuilder;
+
+    /** @var \Magento\Framework\Setup\SchemaSetupInterface $setup */
+    private SchemaSetupInterface $setup;
+
+    /** @var array $replaceValues */
+    private array $replaceValues = [
+        '100',
+        '350',
+    ];
+
+    /** @var $newValue */
+    private $newValue = '200';
```

Comment:
Ik zie bij mij ook in de database dat bijvoorbeeld de staffel 20-50 gram opgeslagen wordt als 50 in de database dus wellicht zou deze waarde ook 350 moeten zijn